Doc: Issue68 add set search_path to env_setup.md

mtreacy002 opened the pull request:
### Description

Add set search_path to ENV_SETUP_INSTRUCTION.md to reflect changes on WIKI page [BIT Development Environment Setup Instructions](https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup)

Fixes #68 

### Type of Change:
<!--**Delete irrelevant options.**-->

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
<!--Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.-->


### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation


**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

mtreacy002 committed:
Issue68 add set search_path to env_setup.md

mtreacy002 commented:
Update @anitab-org/bridgeintech-maintainers . I've updated the WIKI BIT Development Environment Setup Instruction to include setting search_path to bitschema as suggested on our mentors meeting last night. This PR is to make sure the ENV_SETUP_INSTRUCTION.md on .github folder is reflecting the changes. Can you please review and approve this? Thanks

meenakshi-dhanani commented on .github/ENV_SETUP_INSTRUCTION.md:
```diff
@@ -51,15 +50,42 @@ To do this, click on `bit_schema` on the left sidebar, go to `Properties` (top n
+1. Run the next command to show the existing search_path
+> $ psql -c 'show search_path;' -U postgres -d bit_schema_test
+
+// TO DO: ADD AN IMAGE SHOWING THE INITIAL SEARCH PATH. IT SHOULD ONLY HAVE the `public` schema
```
Why is there a TODO comment here?

meenakshi-dhanani commented:
I'm not sure why TODO comments are committed, if you have something to say to another contributor who should add images, please mention the line number to them in the PR, I would not prefer comments like this

mtreacy002 commented:
Ok, I'll remove them both in Wiki link and the doc then'll  push the code change. 👍

mtreacy002 committed:
PR#69 removes TO DO comments

mtreacy002 commented:
Update @meenakshi-dhanani and @anitab-org/bridgeintech-maintainers . I have removed the TO DO comments and replaced with line numbers on relevant issues description.

foongminwong reviewed (approved):
reviewed and looks good to me
